backtrace-library/src/main/java/backtraceio/library/BacktraceCredentials.java

```diff
@@ -76,4 +76,15 @@ public Uri getSubmissionUrl(){
         }
         return getServerUrl();
     }
+
+    public Uri getMinidumpSubmissionUrl() {
+        Uri backtraceJsonUri = getSubmissionUrl();
+        String jsonUrl = backtraceJsonUri.toString();
+        if(jsonUrl.contains("format=json")) {
```

Comment:
Please fix formatting here and in other files

Reply:
done